src/tool_help.c

```diff
@@ -272,6 +272,7 @@ static const char *const helptext[] = {
   "     --tlspassword STRING  TLS password",
   "     --tlsauthtype STRING  TLS authentication type (default: SRP)",
   "     --unix-socket FILE    Connect through this Unix domain socket",
+  "     --abstract-unix-socket PATH Connect to an abstract Unix domain socket",
```

Comment:
It isn't a file in case of abstract, but path would be fine for both, I'll fix it.